test/test_math_ops.cpp

```diff
@@ -671,6 +671,57 @@ TEST(MathOps, FloorModBroadcasting) {
   opexecuter.RunTest();
 }  // end of test op FloorModBroadcasting
 
+
+// Test op: FloorDivNegInt
+// Error found when running tensorflow python test
+// should enable when NGraph fix the issue
+TEST(MathOps, DISABLED_FloorDivNegInt) {
+  Scope root = Scope::NewRootScope();
+
+  Tensor A(DT_INT32, TensorShape({1}));
+  Tensor B(DT_INT32, TensorShape({1}));
+
+  vector<int> A_vect{-1}; 
+  vector<int> B_vect{3}; 
+
+  AssignInputValues(A, A_vect);
+  AssignInputValues(B, B_vect);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::FloorDiv(root, A, B);
+
+  vector<DataType> output_datatypes = {DT_INT32};
+
+  std::vector<Output> sess_run_fetchoutputs = {R};
+  OpExecuter opexecuter(root, "FloorDiv", static_input_indexes,
+                        output_datatypes, sess_run_fetchoutputs);
+
+  opexecuter.RunTest();
+}  // end of test op FloorDivNegInt
+
+TEST(MathOps, FloorDivNegFloat) {
+  Scope root = Scope::NewRootScope();
+
+  Tensor A(DT_FLOAT, TensorShape({1}));
+  Tensor B(DT_FLOAT, TensorShape({1}));
+
+  vector<float> A_vect{-1.f}; 
+  vector<float> B_vect{3.f}; 
+
+  AssignInputValues(A, A_vect);
+  AssignInputValues(B, B_vect);
+
+  vector<int> static_input_indexes = {};
+  auto R = ops::FloorDiv(root, A, B);
+
+  vector<DataType> output_datatypes = {DT_FLOAT};
```

Comment:
If inputs are float, out of floordiv is float, but if they are int, output is int? That is interesting, because usually functions like say `std::floor` always output float/double for both int and float inputs.

Do we also want to add a mixed test? A is int, B is float?

Reply:
I think for all Integer input case, the expected output is also Integer. So current NGraph calculation was not right for this case. 

Yes, agree, I will add a mixed test case as well.